packages/injected-script/main/WagmiBridge/config.ts

```diff
@@ -0,0 +1,18 @@
+import type { Chain, FallbackTransport, PublicClient } from 'viem'
+import { arbitrum, aurora, avalanche, bsc, fantom, gnosis, mainnet, optimism, polygon } from 'viem/chains'
+import { configureChains, createConfig, type Config, type WebSocketPublicClient } from '@wagmi/core'
+import { publicProvider } from '@wagmi/core/providers/public'
```

Comment:
don't import them at the top level. import them with dynamic import.

Reply:
I will do it after everything works fine.